mollie/api/resources/subscription_payments.py

```diff
@@ -0,0 +1,20 @@
+from .payments import Payments
+
+
+class SubscriptionPayments(Payments):
+    customer_id = None
+    subscription_id = None
+
+    def get_resource_name(self):
+        return 'customers/{customer_id}/subscription/{subscription_id}/payments'.format(
+            customer_id=self.customer_id,
+            subscription_id=self.subscription_id,
+        )
+
+    def with_parent_id(self, customer_id, subscription_id):
+        self.customer_id = customer_id
+        self.subscription_id = subscription_id
+        return self
+
+    def on(self, subscription):
```

Comment:
As this generates an extra query, we should file a request with Mollie to add the customer id to the response of a subscription.